x/feegrant/CHANGELOG.md

```diff
@@ -35,6 +35,10 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 * [#18047](https://github.com/cosmos/cosmos-sdk/pull/18047) Added a limit of 200 grants pruned per EndBlock and the method PruneAllowances that prunes 75 expired grants on every run.
 
+### Improvements
+
+* (feegrant) [#18767](https://github.com/cosmos/cosmos-sdk/pull/18767) Ensure we only execute revokeAllowance if there is no error is the grant is to be removed.
```

Comment:
Nit, you can remove (feegrant) as it is the changelog is already scoped to feegrant